README.md

```diff
@@ -59,40 +59,44 @@ Scripts for updating and debugging Kafka can be found [here](https://github.com/
+| KAFKA_SEC_CA_CERTS           | _unset_                                                                                                                                                                                                   | PEM [2] of CA cert chain if using private CA for the server cert [1]                                 |
+| KAFKA_SEC_SKIP_VERIFY        | `false`                                                                                                                                                                                                   | ignore server certificate issues if set to `true` [1]                                                |
+| GENERATE_DOWNLOADS_TOPIC     | `filter-job-submitted`                                                                                                                                                                                    | The topic to send generate full dataset version downloads to                                         |
+| HEALTHCHECK_INTERVAL         | `30s`                                                                                                                                                                                                     | The time between calling healthcheck endpoints for check subsystems                                  |
+| HEALTHCHECK_CRITICAL_TIMEOUT | `90s`                                                                                                                                                                                                     | The time taken for the health changes from warning state to critical due to subsystem check failures |
+| ENABLE_PRIVATE_ENDPOINTS     | `false`                                                                                                                                                                                                   | Enable private endpoints for the API                                                                 |
+| DISABLE_GRAPH_DB_DEPENDENCY  | `false`                                                                                                                                                                                                   | Disables connection and health check for graph db                                                    |
+| DOWNLOAD_SERVICE_SECRET_KEY  | `QB0108EZ-825D-412C-9B1D-41EF7747F462`                                                                                                                                                                    | A key specific for the download service to access public/private links                               |
```

Comment:
I realise this isn't new but just to confirm - that secret key is not actual key material?

Reply:
To be honest I don't know - they were part of the changes I made and I'm assuming there okay since they must have passed PR previously

Reply:
I just checked the secrets files for both develop and prod and can confirm this is not an actual key value 🙂

Reply:
Thanks Rafa :)